opendcs-rest-api/src/main/java/org/opendcs/odcsapi/appmon/ApiEventClient.java

```diff
@@ -29,39 +29,42 @@
 import org.opendcs.odcsapi.errorhandling.WebAppException;
 import org.opendcs.odcsapi.util.ApiBasicClient;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class ApiEventClient
 	extends ApiBasicClient
 {
+	private static final Logger LOGGER = LoggerFactory.getLogger(ApiEventClient.class);
 	private Long appId = null;
 	private long lastActivity = 0L;
 	private String appName = null;
 	private BufferedReader reader = null;
 	private SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yy HH:mm:ss");
 	private long pid = 0L;
 
-	public void connect() 
-		throws UnknownHostException, IOException
-	{
-System.out.println("Connecting to " + this.host + ":" + this.port);
-		super.connect();
-		reader = new BufferedReader(new InputStreamReader(this.input));
-	}
-	
 	public ApiEventClient(Long appId, String hostname, int port, String appName, long pid)
 	{
 		super(hostname, port);
 		this.appId = appId;
 		this.appName = appName;
 		sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
 		this.pid = pid;
-System.out.println("ApiEventClient appId=" + appId + ", host=" + hostname + ", port=" + port + ", name=" + appName);
-super.debug = System.out;
+		LOGGER.debug("ApiEventClient appId={}, host={}, port={}, name={}", appId, hostname, port, appName);
```

Comment:
Converting these all from system.out debug statements to use slf4j.  They seem like useful debug statements, but let me know if they should not be in here and I can remove them.